packages/stacks-docs/product/components/empty-states.html

```diff
@@ -1,95 +1,76 @@
 ---
 layout: page
 title: Empty states
+svelte: https://beta.svelte.stackoverflow.design/?path=/docs/components-emptystate--docs
+figma: https://www.figma.com/design/do4Ug0Yws8xCfRjHe9cJfZ/Project-SHINE---Product-UI?node-id=610-18807&p=f&m=dev
 description: Empty states are used when there is no data to show. Ideally they orient the user by providing feedback based on the the user’s last interaction or communicate the benefits of a feature. When appropriate, they should explain the next steps the user should take and provide guidance with a clear call-to-action.
 tags: components
 ---
 <section class="stacks-section">
-    {% header "h2", "No data" %}
+    {% header "h2", "No data or results" %}
     <p class="stacks-copy">
-        Used when a user hasn’t interacted with a feature yet.
+        Typical use-case for an empty state is when a feature has no data or a search/filter operation yields no results.
     </p>
-    {% header "h3", "No data actionable" %}
+    {% header "h3", "Actionable" %}
     <p class="stacks-copy">
-        If the user can take an action to fix the situation, it's best to draw attention to it, explain next steps, and provide a call-to-action.
+        If the user is able to address the situation resulting in an empty state, it is appropriate to include a button for them to do so.
     </p>
     <div class="stacks-preview">
 {% highlight html %}
 <div class="s-empty-state wmx4 p48">
-    @Svg.Spot.EmptyXL.With("mb24")
-    <p>Why is this blank? Explain it.</p>
-    <a class="s-btn s-btn__filled">Call to action</a>
+    @Svg.Spot.Empty.With("native")
+    <h4 class="s-empty-state__title">No questions match your result.</h4>
+    <p>Try refining your search term or trying something more general.</p>
+    <button class="s-btn s-btn__filled">Clear filters</a>
```

Comment:
Looks like the mocks use `tonal` buttons.

```suggestion
    <button class="s-btn s-btn__tonal">Clear filters</a>
```